cmake/dependencies/pybind11.cmake

```
@@ -78,9 +78,9 @@ set(openPMD_pybind11_branch "v2.13.6"
     "Repository branch for openPMD_pybind11_repo if(openPMD_USE_INTERNAL_PYBIND11)")
 
 if(openPMD_USE_PYTHON STREQUAL AUTO)
-    find_package(Python 3.7.0 COMPONENTS Interpreter Development.Module)
+    find_package(Python 3.9.0 COMPONENTS Interpreter Development.Module)
```

Since 3.8 is still widely used in Ubuntu 20.04 (system default), we could be chill here:
```suggestion
    find_package(Python 3.8.0 COMPONENTS Interpreter Development.Module)
```